doc/lightning-fundchannel_start.7

```diff
@@ -1,9 +1,9 @@
 .TH "LIGHTNING-FUNDCHANNEL_START" "7" "" "" "lightning-fundchannel_start"
 .SH NAME
-lightning-fundchannel_start - Command for initiating channel establishment for a lightning channel
+
```

Comment:
How did this get corrupted?  Do you need to update your mrkd?

Reply:
I did in fact need to update `mrkd`